packages/graph/hash_graph/lib/graph/src/store/postgres/ontology/entity_type/mod.rs

```diff
@@ -88,29 +100,24 @@ impl<C: AsClient> PostgresStore<C> {
                     }
                 }
 
-                if context.graph_resolve_depths.link_type_resolve_depth > 0
-                    || context.graph_resolve_depths.entity_type_resolve_depth > 0
-                {
-                    let linked_uris = entity_type
-                        .inner()
-                        .link_type_references()
-                        .into_iter()
-                        .map(|(link_type_id, entity_type_ids)| {
-                            (
+                // TODO: Use relation tables
+                //   see https://app.asana.com/0/0/1202884883200942/f
+                for (link_type_id, entity_type_ids) in entity_type.inner().link_type_references() {
+                    edges.insert(
+                        GraphElementIdentifier::OntologyElementId(entity_type_id.clone()),
+                        OutwardEdge {
+                            edge_kind: EdgeKind::References,
+                            destination: GraphElementIdentifier::OntologyElementId(
                                 link_type_id.clone(),
-                                entity_type_ids
-                                    .iter()
-                                    .map(|reference| reference.uri().clone())
-                                    .collect::<Vec<_>>(),
-                            )
-                        })
-                        .collect::<Vec<_>>();
+                            ),
+                        },
+                    );
 
-                    // TODO: Use relation tables
-                    //   see https://app.asana.com/0/0/1202884883200942/f
-                    for (link_type_id, entity_type_ids) in linked_uris {
+                    if context.graph_resolve_depths.link_type_resolve_depth > 0
+                        || context.graph_resolve_depths.entity_type_resolve_depth > 0
```

Comment:
I think we don't need these anymore (I added them to avoid iterate over the link type references if not needed)